pkg/lang/csharp/queries/expose/map_route.scm

```diff
@@ -0,0 +1,11 @@
+(invocation_expression
+  function:
+  (member_access_expression expression: (_) @var
+    name: (_) @method_name (#match? @method_name "^Map($|Get|Post|Put|Delete)$")
```

Comment:
Does the `($|` case work with the second final `)$`? Ie, is this the same as `($|(Get|Post|Put|Delete)$)`

Reply:
probably not. I'll change that.